libs/pulumi-static-site/README.md

```diff
@@ -0,0 +1,23 @@
+# pulumi-static-site
+
+Provides an S3 bucket behind a CloudFront distribution for use as a static site.
```

Comment:
Is file upload and such out of scope? IMO it's the harder problem to solve, if you upload the index file before all the other assets you break your website.

Creating a bucket with a distribution is the easier half of the problem I think. 

Also still leaves the cleanup of assets and stuff (which is hard) to the user, further reducing the value of the module.

I wonder if a blog post on the approach would be higher value than a module like this?

Reply:
It's in scope if we want to continue to support shorthand uploads. Can descope and continue to use the existing terraform template.

Reminder that the scope of this task is to implement the existing terraform template in pulumi.

Reply:
I think if those are the goals, probably should make it a private module like the sumo forwarder